package-lock.json

```diff
@@ -19770,7 +19775,7 @@
 				},
 				"node-pre-gyp": {
 					"version": "0.12.0",
-					"resolved": "https://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.12.0.tgz",
+					"resolved": false,
```

Comment:
These values keep being flipped back and forth. I summarized some findings at https://github.com/WordPress/gutenberg/issues/16157#issuecomment-599589261 . Out of curiosity, do you use Linux for development (or another OS which would "opt out" of the optional dependencies here)? Or an older version of NPM (older than latest `6.14.2`)?

Reply:
I'm now on npm `6.14.2`, I believe I was on `6.14.1` at the time of this commit. I'm using macOS all the time. I have no idea why this value changes so often.

Reply:
I strongly suspect it's related to the fact that the dependencies are "optional", though it seems like in macOS they would not typically be skipped. In any case, it does feel like it's something largely on `npm`'s end (i.e. I do not think it's correct that this would ever be `false`). Wondering if we could do something to address it in the meantime though. For example, we could create a script which is run during the build and verifies deeply that there are no `"resolved": false` in `package-lock.json`. But I'm not sure yet how we make these issues actionable from the perspective of the pull request author; in other words, how it would be expected that you would fix the problem.